cpp/cppsdk/gsdk.cpp

```diff
@@ -355,6 +355,41 @@ namespace Microsoft
                         }
                     }
 
+                    if (heartbeatResponse.isMember("maintenanceSchedule"))
```

Comment:
wondering if this entire block should come with a try / catch block. To catch for JSON exceptions. If this is already taken care of by an outer try / catch that I am not spotting now, please disregard.

Reply:
there's an existing try catch for this, yes